src/index.tsx

```diff
@@ -1,59 +1,15 @@
-import React from 'react';
-import { Provider } from 'react-redux';
+import React from "react";
+import { render, unmountComponentAtNode } from "react-dom";
+import App from "./App";
 
-import Widget from './components/Widget';
+const MOUNT_NODE = document.getElementById("root");
```

Comment:
Let's make it something like `dancecard-leadnurturing-chatbot` so that we don't forget.